nebula_decoders/src/nebula_decoders_velodyne/decoders/vlp16_decoder.cpp

```diff
@@ -237,13 +237,20 @@ void Vlp16Decoder::unpack(const velodyne_msgs::msg::VelodynePacket & velodyne_pa
 
               // Condition added to avoid calculating points which are not in the interesting
               // defined area (min_angle < area < max_angle).
-              if (
+              const bool is_within_min_max_angle =
                 (azimuth_corrected >= sensor_configuration_->cloud_min_angle * 100 &&
                  azimuth_corrected <= sensor_configuration_->cloud_max_angle * 100 &&
-                 sensor_configuration_->cloud_min_angle < sensor_configuration_->cloud_max_angle) ||
+                 sensor_configuration_->cloud_min_angle < sensor_configuration_->cloud_max_angle);
+
+              const bool is_outside_max_min_angle =
                 (sensor_configuration_->cloud_min_angle > sensor_configuration_->cloud_max_angle &&
                  (azimuth_corrected <= sensor_configuration_->cloud_max_angle * 100 ||
-                  azimuth_corrected >= sensor_configuration_->cloud_min_angle * 100))) {
+                  azimuth_corrected >= sensor_configuration_->cloud_min_angle * 100));
```

Comment:
Suggestion: as mentioned in the WG meeting, it would be nice to encapsulate this code into a filter class to make it clearer what function is being performed in the decoder, and also reduce repeated code in each decoder.

Reply:
Sorry this comment is on the wrong part of the code, a mistake on my part! Will delete